server/src/test/java/org/elasticsearch/common/joda/SimpleJodaTests.java

```diff
@@ -262,49 +262,6 @@ public void testThatEpochsCanBeParsed() {
         assertThat(formatter.parseJoda(epochFloatValue).getMillis(), is(dateTime.getMillis()));
     }
 
-    public void testThatNegativeEpochsCanBeParsed() {
```

Comment:
Why is this test removed? These would still be parsed by Joda, so would issue deprecation warnings? Or am I misunderstanding something?

Reply:
I added it back with a warning check in efd290b